tests/satellite_constellations/test_satellites.py

```diff
@@ -10,10 +10,10 @@ class TestSatellites(unittest.TestCase):
     def test_constellations(self):
         """Test stellite constellations"""
 
-        mjd0 = survey_start_mjd()
+        mjd0 = SURVEY_START_MJD
         sv1 = starlink_tles_v1()
-        sv2 = starlink_tles_v2()
-        ow = oneweb_tles()
+        _ = starlink_tles_v2()
+        _ = oneweb_tles()
```

Comment:
Rather than set the other tles to `_`, I would just loop the test over [sv1, sv2, ow]. Or if that's slow, check that those functions return something reasonable with something like `assert sv2 is not None`.

Reply:
Good suggestions, will update.